pkgs/kubernetes/kubectl/pkg.yaml

```diff
@@ -1,6 +1,5 @@
 packages:
   - name: kubernetes/kubectl@v1.25.3
-  - name: kubernetes/kubectl
-    version: v1.22.15
-  - name: kubernetes/kubectl
-    version: v1.20.15
+  - name: kubernetes/kubectl@v1.24.7
+  - name: kubernetes/kubectl@v1.23.13
+  - name: kubernetes/kubectl@v1.22.15
```

Comment:
```suggestion
  - name: kubernetes/kubectl
    version: v1.22.15
```

We test `version_overrides` with multiple versions.
So the number of versions should be same as `version_constraint`.